cargo-nextest/src/dispatch.rs

```diff
@@ -762,6 +765,57 @@ impl TestBuildFilter {
     }
 }
 
+#[derive(Debug, Args)]
+#[command(next_help_heading = "Filter options")]
+struct ArchiveBuildFilter {
+    /// Test filterset (see {n}<https://nexte.st/docs/filtersets>).
+    #[arg(
+        long,
+        alias = "filter-expr",
+        short = 'E',
+        value_name = "EXPR",
+        action(ArgAction::Append)
+    )]
+    filterset: Vec<String>,
+
+    /// Ignore the default filter configured in the profile.
+    ///
+    /// By default, all filtersets are intersected with the default filter configured in the
+    /// profile. This flag disables that behavior.
+    ///
+    /// This flag doesn't change the definition of the `default()` filterset.
+    #[arg(long)]
+    ignore_default_filter: bool,
```

Comment:
I don't think we should support `ignore_default_filter` -- unless you see a reason to do so?